python-aexpect.spec

```diff
@@ -34,11 +34,12 @@ sftp, telnet, among others.
 
 %install
 %{__python} setup.py install --root %{buildroot} --skip-build
+mv %{buildroot}%{_bindir}/aexpect-helper %{buildroot}%{_bindir}/aexpect-helper-%{python_version}
```

Comment:
Looks good, I now understand the remarks on Fedora/EPEL packaging.  Please hold https://src.fedoraproject.org/rpms/python-aexpect/pull-request/1 until we get this fixed.

Reply:
Of course. I was planning to hold the PR until a new release of `aexpect` was formally tagged.